recipes/atari-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "atari-py" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6249ad5079b0489e87eb44e65485bb1b07cc1b5af729f1ee52ece749503ceb1d
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - pip
+    - python
+    - zlib
+  run:
+    - python
+    - numpy
+    - six
+    - zlib
+
+test:
+  imports:
+    - atari_py
+  requires:
```

Comment:
If you're not running the test suite, you don't need `nose2` or the `requires` section